tools/lint-hotkeys

```diff
@@ -39,7 +39,7 @@ def lint_hotkeys_file() -> None:
     existing OUTPUT_FILE
     """
     hotkeys_file_string = get_hotkeys_file_string()
-    error_flag = 0 if is_every_key_category_valid() else 1
+    error_flag = not is_every_key_category_valid()
```

Comment:
NIT: As it's now a boolean, it'd also be useful to have a cleaner name for this.

Maybe `is_any_error_present`, or similar.